community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/CypherAdapterStream.java

```diff
@@ -152,24 +150,26 @@ public static AnyValue convert( Iterable<Notification> notifications )
             List<AnyValue> out = new ArrayList<>();
             for ( Notification notification : notifications )
             {
-                Map<String,AnyValue> notificationMap = new HashMap<>( 4 );
-                notificationMap.put( "code", stringValue( notification.getCode() ) );
-                notificationMap.put( "title", stringValue( notification.getTitle() ) );
-                notificationMap.put( "description", stringValue( notification.getDescription() ) );
-                notificationMap.put( "severity", stringValue( notification.getSeverity().toString() ) );
+                MapValueBuilder builder = new MapValueBuilder();
```

Comment:
```new MapValueBuilder(4);```

Reply:
I'm torn in this case it is mostly four but in some cases if `includePosition` is true we include a fifth element, so maybe go with size 5 instead.